cfpb/cfgov-refresh
# Ask CFPB: Fix for answer page analytics event

Send category name instead of slug in Ask answer page `Virtual Pageview` event

## Testing

1. Check out branch
2. Run `gulp scripts`
3. Navigate to [Spanish](http://localhost:8000/es/obtener-respuestas/que-efecto-tendria-sobre-mi-credito-comparar-diferentes-ofertas-de-prestamos-para-vehiculos-es-763/) and [English](http://localhost:8000/ask-cfpb/what-effect-will-shopping-for-an-auto-loan-have-on-my-credit-en-763/) answer pages and inspect the `dataLayer` object in console. The `category` property on the `Virtual Pageview` event should be the answer category's name instead of slug.

## Screenshots

<img width="579" alt="screen shot 2017-06-19 at 10 04 17 am" src="https://user-images.githubusercontent.com/778171/27296913-b6b06ab4-54d7-11e7-86cc-54dddecb4837.png">
<img width="518" alt="screen shot 2017-06-19 at 10 04 48 am" src="https://user-images.githubusercontent.com/778171/27296914-b6b0cb08-54d7-11e7-8fd0-f8ff7ea92659.png">


## Checklist
